### PR TITLE
Wasm release banner

### PR DESCRIPTION
**Pull request by sirius651**
**Pull Request Summary**

> Add warm release banner on dapp staking page

**Check list**

- [ ] contains breaking changes
- [x] adds new feature
- [ ] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

스크린샷 2023-03-24 오후 12 33 51

### CODE REVIEW

**Comment by Kahonnohak** [comment]
@sirius651 perfect thank you nothing I need to add for this PR apart from - if you could add a dummy link to the image I can replace it later.

**Comment by sirius651** [comment]
> @sirius651 perfect thank you nothing I need to add for this PR apart from - if you could add a dummy link to the image I can replace it later.

Added

File: src/components/dapp-staking/my-staking/WasmAdArea.vue [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<template>
+  <div class="wrapper--card">
+    <div
+      class="wrapper--img"
+      :style="`background-image: url('${bg_img.wasm_hero}')`"
+      @click="goToLink()"
```
Comment (impelcrypto) [inline src/components/dapp-staking/my-staking/WasmAdArea.vue:6]: Could you advise where to click to open the docs page? I've clicked on the marked area but nothing has happened.

image

Comment (impelcrypto) [reply src/components/dapp-staking/my-staking/WasmAdArea.vue:6]: image

Comment (sirius651) [reply src/components/dapp-staking/my-staking/WasmAdArea.vue:6]: of course, WASM ads area. it seems working well on my side. can you clear cache and trying again?

Comment (Kahonnohak) [reply src/components/dapp-staking/my-staking/WasmAdArea.vue:6]: Yes I have checked and looks good thanks! I'll update as soon as I heard things from the team.

Comment (impelcrypto) [reply src/components/dapp-staking/my-staking/WasmAdArea.vue:6]: @sirius651 Ok, this is interesting. It works on Chrome but not on FireFox.

Comment (impelcrypto) [reply src/components/dapp-staking/my-staking/WasmAdArea.vue:6]: > It works on Chrome but not on FireFox.

It worked after clearing cache on FireFox.

**Review by impelcrypto — APPROVED** [review]

File: src/components/dapp-staking/my-staking/WasmAdArea.vue (hunk 1 above)
Comment (sirius651) [reply src/components/dapp-staking/my-staking/WasmAdArea.vue:6]: Hm which version of FF are you using? In my FF, it works well.

Comment (impelcrypto) [reply src/components/dapp-staking/my-staking/WasmAdArea.vue:6]: 111.0 (64-bit)

image

**Review by bobo-k2 — APPROVED** [review]

**Pull request merged**